docs/guides/tutorials/nextjs_app_router.rst

```diff
@@ -0,0 +1,627 @@
+.. _ref_guide_nextjs_app_router:
+
+====================
+Next.js (App Router)
+====================
+
+:edb-alt-title: Building a simple blog application with
+   EdgeDB and Next.js (App Router)
+
+We're going to build a simple blog application with
+`Next.js <https://nextjs.org/>`_ and EdgeDB. Let's start by scaffolding our
+app with Next.js's ``create-next-app`` tool.
+
+.. code-block:: bash
+
+  $ npx create-next-app@latest
+
+You'll be prompted to provide a name (we'll use ``nextjs-blog``) for your 
+app and choose project options. For this tutorial, we'll go with the 
+recommended settings including TypeScript, App Router, and 
+**opt-ing out** of the ``src/`` directory.
+
+The scaffolding tool will create a simple Next.js app and install its 
+dependencies. Once it's done, you can navigate to the app's directory and 
+start the development server.
+
+.. code-block:: bash
+
+  $ cd nextjs-blog
+  $ yarn dev # or pnpm dev or npm run dev or bun dev
+
+Open `localhost:3000 <http://localhost:3000>`_ to see the default Next.js
+homepage. At this point the app's file structure looks like this:
```

Comment:
If you already have something running on 3000, the dev server will listen on a different port. Does it matter, or should we assume that, if you're running something on 3000, you'll probably be able to correct for that?

Maybe we just say something like this to avoid the issue:
> When the dev server starts, it will log out a local URL. Visit that URL to see the default Next.js homepage.

Reply:
Makes sense.